skyrimnet-chatterbox/shared_config.py

```
@@ -0,0 +1,207 @@
+    """
+    language = language.lower()
+    if language is None:
+        normalized = "en"
+    elif language in ["zh-cn", "zh", "cn"]:
+        normalized = "zh-cn"
+    else:
+        normalized = language.split("-")[0] if language else "en"
+    
+    if normalized not in SUPPORTED_LANGUAGE_CODES:
+        raise ValueError(f"Language '{language}' not supported. "
+                        f"Supported languages: {SUPPORTED_LANGUAGE_CODES}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)